pkg/bootconfig/manifest.go

```diff
@@ -1,52 +0,0 @@
-// Copyright 2017-2019 the u-root Authors. All rights reserved
```

Comment:
what is this manifest stuff? unused by systemboot, too?

Reply:
Yep it was only used by us